docker/docker-entrypoint.d/30-sed-on-appconfig.sh

```diff
@@ -43,10 +43,10 @@ if [ -n "${APP_CONFIG_OAUTH2_CLIENTID}" ]; then
     -i "$APP_CONFIG_FILE"
 fi
 
-if [ -n "${APP_CONFIG_OAUTH2_IMPLICIT_FLOW}" ]; then
-  echo "SET APP_CONFIG_OAUTH2_IMPLICIT_FLOW"
+if [ -n "${APP_CONFIG_OAUTH2_CODE_FLOW}" ]; then
```

Comment:
this is a breaking change, the implicit flow should still be supported I believe